libraries/Microsoft.Bot.Builder.Skills/BotFrameworkSkill.cs

```diff
@@ -3,7 +3,7 @@
 
 using System;
 
-namespace Microsoft.Bot.Builder.Integration.AspNet.Core
+namespace Microsoft.Bot.Builder.Integration.AspNet.Core.Skills
```

Comment:
Is this supposed to be:

```cs
namespace Microsoft.Bot.Builder.Skills
```

Reply:
I am keeping it in the same namespace as the others for now. We can move it to Microsoft.Bot.Builder.Skills this week if we agree on it.